### PR TITLE
Additional work for fixDropdowns

### PR DESCRIPTION
**Pull request by mchlcrtz**
## Description
Adjust dropdown/selectInput to clear selected option 

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [x] Code compiles successfully (verified via `yarn start`)
- [x] No lint issues exist (verified via `yarn lint`)
- [x] New and existing unit tests pass (verified via `yarn test`)
- [x] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-info-components/src/Form/SelectInput/index.js [hunk 1]
```diff
@@ -22,16 +21,19 @@ class SelectInputContainer extends React.PureComponent {
   }
 
   handleChange (item) {
+    // when cleared
+    // if item value is null, clear values
+    // otherwise, set state to chosen value
+    const value = item === null ? undefined : item.value
```
Comment (plondon) [inline packages/blockchain-info-components/src/Form/SelectInput/index.js]: Comment is unnecessary here as it's clear from the ternary what's going on.

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoin/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { connect } from 'react-redux'
 import styled from 'styled-components'
 import SelectBox from '../SelectBox'
 import { Icon, Text } from 'blockchain-info-components'
-import { prop } from 'ramda'
+// import { prop } from 'ramda'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoin/index.js]: Comment

**Comment by plondon** [comment]
I added some more info about whats needed here: https://github.com/blockchain/blockchain-wallet-v4-frontend/issues/843

File: packages/blockchain-info-components/src/Form/SelectInput/template.js [hunk 3]
```diff
@@ -49,11 +49,12 @@ const SelectInput = props => {
   } = props
   const options = items.map(item => ({ value: item.value, label: item.text }))
   const defaultValue = head(filter(x => equals(x.value, defaultItem), options))
-
+  // Give StyledSelect component isClearable prop
```
Comment (plondon) [inline packages/blockchain-info-components/src/Form/SelectInput/template.js]: This is also unnecessary to comment in the code itself, but maybe you want to comment on the PR?

Comment (mchlcrtz) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js]: Thank you for the feedback. I will amend these commits and continue to work on #843

File: packages/blockchain-info-components/src/Form/SelectInput/index.js [hunk 4]
```diff
@@ -22,16 +21,16 @@ class SelectInputContainer extends React.PureComponent {
   }
 
   handleChange (item) {
+    const value = item === null ? undefined : item.value
```
Comment (tony-blockchain) [inline packages/blockchain-info-components/src/Form/SelectInput/index.js]: `prop('value', item)` would work best here (`prop` from `ramda`)

**Comment by plondon** [comment]
@mchlcrtz sorry for the delay on this review. Why do we need the isClearable option? The header option for Exchange can't be selectable, let me know if you need help with that one.

**Pull request merged**